plugins/builds.js

```diff
@@ -100,7 +88,7 @@ exports.register = (server, options, next) => {
                 const size = Buffer.byteLength(request.payload);
 
                 if (username !== buildId) {
-                    return reply(boom.forbidden(`Credential only valid for ${username}`));
+                    return boom.forbidden(`Credential only valid for ${username}`);
```

Comment:
Also in the doc, seems like you can just `throw boom.xxxxx`. what's the difference between that and `return`?

Reply:
After reviewing the docs, it seems there isn't a difference. I'll change all error responses to be sent via `throw` instead of `return`. Nice catch 👍